src/Templates/src/templates/maui-blazor/MauiApp.1.csproj

```diff
@@ -60,6 +60,9 @@
         <PackageReference Include="Microsoft.Extensions.Logging.Debug" Version="MS_EXT_VERSION" />
     </ItemGroup>
 
+  <!-- Build Properties must be defined within these property groups to ensure successful publishing
+       to the App Store. See "Define build properties in your project file" here:
+       https://learn.microsoft.com/en-us/dotnet/maui/mac-catalyst/deployment/publish-app-store -->
```

Comment:
1. This needs to be an aka.ms link
2. Make sure you remove the `en-us` from the URL so that people don't always go to the English-US docs. If you omit the `en-us` then it will auto-redirect to the docs in the user's preferred locale

Reply:
(Same applies to all other links.)